src/Neo.VM/Utility.cs

```diff
@@ -61,30 +60,18 @@ public static BigInteger Sqrt(this BigInteger value)
             return z;
         }
 
-#if !NET5_0_OR_GREATER
-        static int GetBitLength(this BigInteger i)
+        /// <summary>
+        /// Gets the number of bits required for shortest two's complement representation of the current instance without the sign bit.
+        /// </summary>
+        /// <returns>The minimum non-negative number of bits in two's complement notation without the sign bit.</returns>
+        /// <remarks>This method returns 0 if the value of current object is equal to <see cref="BigInteger.Zero"/> or <see cref="BigInteger.MinusOne"/>. For positive integers the return value is equal to the ordinary binary representation string length.</remarks>
+        public static long GetBitLength(this BigInteger value)
         {
-            byte[] b = i.ToByteArray();
-            return (b.Length - 1) * 8 + BitLen(i.Sign > 0 ? b[b.Length - 1] : 255 - b[b.Length - 1]);
-        }
+            if (value.IsZero) return 0;
+            if (value.GetByteCount(false) > 4096) // Integer.MaxSize = 32
```

Comment:
What are you doing here? This isn't going to solve anything. You need to remove it
```suggestion
```

So your telling me i have this number?
![image](https://github.com/neo-project/neo/assets/8141309/0690c0d0-e7e9-4622-9434-f5ac90bc9d6f)
![image](https://github.com/neo-project/neo/assets/8141309/7f623efb-ab81-40cf-a029-64f7a278bc35)